doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -0,0 +1,100 @@
+.. _using_zephyr_as_uos:
+
+Using Zephyr as User OS
+#######################
+
+The tutorial describes the steps to run zephyr as User OS on ACRN hypervisor. We are using
+Kaby Lake-based NUC model NUC7i5DNHE and you can try other platform ACRN supported as well.
+
+Introduction to Zephyr
+**********************
+
+The Zephyr is a scalable real-time operating-system (RTOS) supporting multiple hardware architecture,
+optimized for resource constrained devices, and built with safety and security in mind.
+
+Steps for Using Zephyr as User OS
+*********************************
+
+#. Build Zephyr
+
+   First of all, please follow the `Getting Started Guide
+   <https://docs.zephyrproject.org/latest/getting_started/>`_ to
+   setup Zephyr development environment. Then follow the `Board-Specfic Support
+   <https://docs.zephyrproject.org/latest/boards/index.html>`_ to build the acrn supported zephyr image ``zephyr.elf``.
+   
+   .. note::
+      When the `Getting Started Guide` for building acrn supported zephyr image got merged to `Zephyr Tree
+      <https://github.com/zephyrproject-rtos/zephyr>`_, this section will be update to the exact reference.
+  
+#. Build grub2 boot loader image
+
+   We can build grub2 bootloader for zephyr using ``boards/x86/common/scripts/build_grub.sh``
+   which locate in `Zephyr Sourcecode <https://github.com/zephyrproject-rtos/zephyr>`_.
+
+   .. code-block:: none
+
+      $ ./boards/x86/common/scripts/build_grub.sh x86_64
+
+   Find the binary at boards/x86/common/scripts/grub/bin/grub_x86_64.efi.
+
+#. Preparing the boot device
+
+   .. code-block:: none
+
+      $ dd if=/dev/zero of=zephyr.img bs=1M count=1024
```

Comment:
This creates a 1GB image, and we don't even need 1.5M with all the files. Let's create a smaller one:
```suggestion
      $ dd if=/dev/zero of=zephyr.img bs=1M count=35
```

**Note:** I am using 35MB on purpose here as this is the smallest partition size allowed by `FAT32`.

Reply:
Yes, will do.